tier4_localization_msgs/srv/PoseWithCovarianceStamped.srv

```diff
@@ -1,4 +1,5 @@
 geometry_msgs/PoseWithCovarianceStamped pose_with_covariance
 ---
 bool success
+bool reliability
```

Comment:
If it's a boolean, I think "reliable" would be better. What do you think? With "reliability," it might be mistaken for a continuous value like a float.

Reply:
rename `reliability` to `reliable` https://github.com/tier4/tier4_autoware_msgs/pull/143/commits/e60fa6892ea7901316412da297c87e482443558b